frontend/src/components/Contributions/ContributionListItem.vue

```diff
@@ -54,7 +54,7 @@
                 v-b-toggle="collapsId"
                 icon="chat-dots"
                 class="h2 mr-5"
-                @click="getListContributionMessages"
+                @click="$emit('closeAllOpenCollapse'), getListContributionMessages"
```

Comment:
should the emit not be part of `getListContributionMessages`? I am asking because I do not know the correct standards of vue.